### ISSUE TITLE
Really need a random next video function

### ISSUE THREAD

**Issue by BMania1999** [vzhd1701/gridplayer#197]
### Suggestion

I notice there is a [Repeat: Directory Random] function,
but I can't manually do it for each or all video.

Could you add a [Playback: Next Random Video Directory]??

Thanks a lot

**Comment by Bindkedon** [comment]
Yes, a setting that forces the next video to play at end of the current one would be fine.
